### ISSUE TITLE
ork file causes stack overflow

### ISSUE THREAD

**Issue by JoePfeiffer** [openrocket/openrocket#1778]
User reports importing rocksim file, after some editing OR crashes.  The user wound up with the attached .ork file, which causes a stack overflow (and no stack trace) when I try to load it.  See https://www.rocketryforum.com/threads/error-in-openrocket.175614/?utm_source=newsletter&utm_medium=email&utm_campaign=2022-10-31&utm_content=daily_newsletter#post-2339011
[LOC-75 edited.ork.txt](https://github.com/openrocket/openrocket/files/9901767/LOC-75.edited.ork.txt)

**Comment by SiboVG** [comment]
The issue is caused by the parachute. If you delete that component from the .ork, the error disappears. Still searching for the exact cause.

**Comment by SiboVG** [comment]
*ding, ding* found it. The culprit is the following line from the parachute's XML:
```XML
<preset type="PARACHUTE" manufacturer="LOC/Precision" partno="LP-78" digest="eb283078de76260abab77feb91810714"/>
```

**Comment by SiboVG** [comment]
Okay, the issue was with the LP-78 parachute preset. There was a bug on importing parachutes (#1723), which was solved in #1726. But the issue is already solved, so closing this.
